src/DotNetBoilerplate.Infrastructure/DAL/Handlers/Events/GetAllEventsHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿using DotNetBoilerplate.Application.Events;
+using DotNetBoilerplate.Application.Events.DTOs;
+using DotNetBoilerplate.Application.Events.Responses;
+using DotNetBoilerplate.Infrastructure.DAL.Contexts;
+using DotNetBoilerplate.Shared.Abstractions.Queries;
+using Microsoft.EntityFrameworkCore;
+
+namespace DotNetBoilerplate.Infrastructure.DAL.Handlers.Events;
+
+internal sealed class GetAllEventsHandler(
+    DotNetBoilerplateReadDbContext dbContext
+)
+    : IQueryHandler<GetAllEventsQuery, EventsInfoResponse>
+{
+    public async Task<EventsInfoResponse> HandleAsync(GetAllEventsQuery query)
+    {
+        var events = await dbContext.Events
+            .AsNoTracking() 
+            .Select(e => new EventDto
+                {
+                    Id = e.Id,
+                    Title = e.Title,
+                }
+            )
+            .ToListAsync(); 
+        
+        return new EventsInfoResponse(events);
```

Comment:
Trzymajmy się tylko nazwenictwa GetAllEventsHandler, GetAllEventsQuery to możemy sobie nazwać ten response GetAllEventsResponse